backend/tests/integration/orcid/test_orcid_domain_models.py

```diff
@@ -467,26 +468,39 @@ def test_happy_flow_put(self):
             self.factory.record_metadata.json["dois"] = dois
             self.factory_clashing.record_metadata.json["deleted"] = True
 
-            pusher = domain_models.OrcidPusher(self.orcid, self.recid, self.oauth_token)
-            result_putcode = pusher.push()
-
-        assert self.putcode == result_putcode
-        assert not self.cache.has_work_content_changed(self.inspire_record)
+            with pytest.raises(exceptions.DuplicatedExternalIdentifierPusherException):
+                tasks.orcid_push.apply_async(
+                    queue="orcid_push_legacy_tokens",
+                    kwargs={
+                        "orcid": self.orcid,
+                        "rec_id": self.recid,
+                        "oauth_token": self.oauth_token,
+                    },
+                    throw=True,
+                )
```

Comment:
what about calling the push again here manually, so the happy flow succeeds (with a comment that celery will do that)? then you don't have to modify the next test.